src/main/java/org/zalando/nakadi/validation/MetadataValidator.java

```diff
@@ -17,6 +17,7 @@
     }
 
     private Optional<ValidationError> checkDateTime(final String occurredAt) {
-        return validator.validate(occurredAt).map(ValidationError::new);
+        return validator.validate(occurredAt)
+                .map(e-> new ValidationError("#/metadata/occurred_at:" + e));
```

Comment:
I think we can improve even further, by improving RFC3339DateTimeValidator error message.